types/react-native/index.d.ts

```diff
@@ -2033,6 +2033,11 @@ export interface ViewPropsAndroid {
      * re-used and re-composited with different parameters. The downside is that this can use up limited video memory, so this prop should be set back to false at the end of the interaction/animation.
      */
     renderToHardwareTextureAndroid?: boolean;
+
+    /**
+     * Whether this `View` should be focusable with a non-touch input device, eg. receive focus with a hardware keyboard.
+     */
```

Comment:
Should we add the platform information to the docs?

Reply:
These are grouped in a ‘ViewPropsAndroid’ interface to indicate they are Android only. It’s not the greatest, but should be enough to ensure people understand when they can use these props?

Reply:
Ah, I haven’t seen that, thanks for the explanation.

I think it should be enough when you look into the actual types, but it always pops up in intellisense and isn’t mentioned in the documentation string, so unless you look into the type definition it’s not directly visible. I think having it mentioned in there might increase the visibility. Not sure though if this is a problem at all for other people or if this is part of a larger discussion.

Reply:
Until we have a way to only load the typings for the platforms one actually targets, I’m not opposed to over-communicating it, so by all means do add it 👍